README.md

```diff
@@ -64,14 +65,15 @@ This project is about building a personal portfolio website - specifically about
 - **[Use-of-HTML5 tools]**
 - **[Follow HTML and CSS best practice]**
 - **[Style your picture placehilders]**
-- **[se of css grid properrty]**
+- **[Use of css grid properrty]**
+- **[Use of HTML form with custamization]**
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](http://127.0.0.1:5500/portfolio.html)
```

Comment:
- [ ] The link may be working on your side because you have the local server running, but it will not work on other people's devices. Since the project is not deployed yet, kindly remove this line and put `Coming Soon...` or some other placeholder text. You can update it after the project is deployed. :+1: